src/my-requests.tsx

```diff
@@ -208,114 +210,180 @@ const ProceedingRow = (props: ProceedingRowProps) => {
 
     const original_request = findOriginalRequest(props.proceeding);
 
+    const initialMessage = useMemo<Omit<Message, 'id'>>(
+        () => ({
+            transport_medium: 'email',
+            date: new Date(),
+            reference: props.proceeding.reference,
+            sentByMe: false,
+            correspondent_address: original_request?.correspondent_address || '',
+            correspondent_email: original_request?.correspondent_email || '',
+            type: 'response',
+        }),
+        [props.proceeding.reference, original_request]
+    );
+
+    const ImportModalContent = () => {
+        const [newMessage, setNewMessage] = useState<Omit<Message, 'id'>>(initialMessage);
+        const addMessage = useProceedingsStore((state) => state.addMessage);
+
+        return (
+            <>
+                <p>
+                    <Text id="import-message-modal-explanation" />
+                </p>
+                <MessageMetadataInput
+                    message={newMessage}
+                    onChange={(message) => setNewMessage(message)}
+                    includeContent={true}
+                />
+                <button
+                    className="button button-secondary"
+                    style="margin-top: 10px;"
+                    onClick={() => {
+                        setNewMessage(initialMessage);
```

Comment:
`confirm()`.

Reply:
why?

Reply:
Clicking this irrevocably discards everything you've entered.

Reply:
but this is pretty clear from the button, isn't it?

Reply:
Destructive actions need confirmation. Deleting an ID data element discards much less work and is similarly obvious, but we still wrap it in a `confirm()`.

Reply:
We could also not delete the data from the state on cancel. What would you prefer?

Reply:
In general, either is fine with me. But if you don't delete the data, I guess you'd need a 'reset' button, which… ugh.